docs/README.md

```diff
@@ -83,7 +71,7 @@ More information about managing your deployments is documented for [Kubernetes](
 * Version specific upgrade paths
 * Collecting system logs through Journald
 
-## Building Agent v2
+## Building
 
```

Comment:
How about "Building the LogDNA Agent" ?? Just the single word "Building" isn't informative enough, imo.

Reply:
The reason I've changed the title is because the table of contents was linking to a `#building` anchor, so I changed the title. I'll change it to "Building the LogDNA Agent" and update the anchor.